datasets/at/meine_abgeordneten/crawler.py

```diff
@@ -17,27 +17,34 @@
 
 def extract_dates(context: Context, url, el):
     active_date_el = el.find('.//span[@class="aktiv"]')
-    inactive_dates_el = el.find('.//span[@class="inaktiv"]')
+    inactive_dates_el = [
+        el.find('.//span[@class="inaktiv"]'),
+        el.find('.//div[@class="text-right ml-auto flex-grow-1"]'),
```

Comment:
I'm trying to think if there's a better way than optionally matching `class="text-right ml-auto flex-grow-1"`. The problem with this is that we won't know when this breaks, at which point, what did we fix? All it takes is for someone to feel like they want a little more padding or margin in there for it to break.

I know it adds complexity, but maybe somehting we can do is this logic:

- if we didn't find an aktiv or an inaktiv
  - assert that el (which is funktionseile) has two children
  - take the first child (this has sources and dates)
  - assert it has three children
  - take the third child (this has dates)
  - take its contents as the date string

if we run that logic on the whole dataset, does it work out ok?